components/atoms/Icon/Icon.js

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import tailwindConfig from '../../../tailwind.config.js'
+import {icons} from './icons'
+
+/**
+ * Convert icon size to px taking rem into account.
+ *
+ * @param {string} size The icon size.
+ * @return {string} The size in px.
+ */
+function sizeToPx(size) {
+  const sizeToRem = {
+    sm: '1',
```

Comment:
Just a thought here, do you think it would be helpful to have icon sizes pre-defined in the tailwind config?

Reply:
I would rather have it here because if we moved it to the config it would have to go in `spacing: {}` and you would be able to do use it where you shouldn't be using it like `padding-icon-sm` or similar.